mesa/cookiecutter-mesa/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+All notable changes to this project will be documented in this file.
```

Comment:
How does this differ from the repo's [history file](https://github.com/projectmesa/mesa/blob/master/HISTORY.rst), and should we maintain both?